src/NHibernate/Engine/QueryParameters.cs

```diff
@@ -201,19 +202,31 @@ public void ValidateParameters()
 
 		public QueryParameters CreateCopyUsing(RowSelection selection)
 		{
-			var copy = new QueryParameters(PositionalParameterTypes, PositionalParameterValues, NamedParameters, LockModes,
-			                               selection, IsReadOnlyInitialized, readOnly, Cacheable, CacheRegion, Comment, CollectionKeys,
-			                               OptionalObject, OptionalEntityName, OptionalId, ResultTransformer)
-			           {
-			           	ProcessedSql = ProcessedSql,
-			            ProcessedSqlParameters = ProcessedSqlParameters != null ? ProcessedSqlParameters.ToList() : null
-			           };
+			var copy = new QueryParameters(
+				PositionalParameterTypes, PositionalParameterValues, NamedParameters, LockModes, selection,
+				IsReadOnlyInitialized, readOnly, Cacheable, CacheRegion, Comment, CollectionKeys, OptionalObject,
+				OptionalEntityName, OptionalId, ResultTransformer)
+			{
+				ProcessedSql = ProcessedSql,
+				ProcessedSqlParameters = ProcessedSqlParameters != null ? ProcessedSqlParameters.ToList() : null,
+				CacheMode = CacheMode
```

Comment:
IMHO it's better to end with "," where allowed.
So it's less change noise when new initialization property is added. And sometimes it even allows to skip merge conflicts.

Reply:
It does not look to me as an usual practice in NHibernate current code base, and while I understand the rationals of it, I still dislike it a bit. So I do not tend to follow this practice.